lib/tasks/dynflow.rake

```diff
@@ -0,0 +1,5 @@
+namespace :dynflow do
```

Comment:
I don't really understand the question. This is the current situation (before merging this PR):

1. In development, by default foreman-tasks & Dynflow run as part of the Rails process. 
2. In development, you can decide to run Dynflow as a separate process by calling `rake dynflow:executor`. Most people don't do this unless it's a git checkout installation, I only knew about this as I wanted to test ActiveJob with the detached executor. It works the same as the one in the web process.
3. In production, this task isn't used, instead `/usr/bin/foreman-tasks start` starts the executor via [systemd](https://github.com/theforeman/foreman-packaging/pull/1549/files#diff-a46acbb74d582e9d3a8ee262a72b237cR11) (and https://github.com/theforeman/foreman-tasks/blob/master/deploy/foreman-tasks.service#L11).

None of 1 2 or 3 are modified after this PR - it works exactly the same. What does the Procfile have to do here? You want to modify the "default" (if you use the Procfile) to run the Dynflow executor as a separate process? Why?